examples/video_upload_multipart.py

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import datetime
+from jwplatform import Client
+from jwplatform.errors import JWPlatformError
+import logging
+import os
+import requests
+import json
+import sys
+import argparse
```

Comment:
Imports need to be reordered.
It also looks like there are a bunch of unused imports (at least `datetime`).